datafusion/proto/src/lib.rs

```diff
@@ -513,7 +513,7 @@ mod roundtrip_tests {
             DataType::FixedSizeBinary(1234),
             DataType::FixedSizeBinary(-432),
             DataType::LargeBinary,
-            DataType::Decimal128(1345, 5431),
+            DataType::Decimal128(123, 234),
```

Comment:
What was the reason for this change?

Reply:
We store precision as a u8 now. 234 fits, 455 doesn't.